fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -316,12 +305,13 @@ protected void configure() {
 
                 bind(getClock()).to(Clock.class);
 
+                DruidDimensionsLoader druidDimensionsLoader = buildDruidDimensionsLoader(
+                        nonUiDruidWebService,
+                        loader.getPhysicalTableDictionary(),
+                        loader.getDimensionDictionary()
+                );
+
                 if (DRUID_DIMENSIONS_LOADER.isOn()) {
-                    DruidDimensionsLoader druidDimensionsLoader = buildDruidDimensionsLoader(
```

Comment:
So that Bank can get a hold of the dimensionDictionary since `builtSegmentMetadata is removed`. Temporary fix...